### ISSUE TITLE
Identify Second use Case

### ISSUE THREAD

**Issue by mheadd** [18F/data-federation-project#128]
## User story
As a member of the U.S. Data Federation, I want to identify a second use case so that we can better understand the challenges behind implementing the tools and guidance offered by the Federation. 
  
## Acceptance criteria
- [ ] Develop a way to measure the magnitude of the challenges faced by the different use cases that we've identified as federated data efforts.
- [ ] Develop a way to evaluate the difficulty of implementing a solution, including a rough estimate of how much time each step of the implementation might take.
- [ ] See if there is a way to ensure we are documenting the impact and difficulty of each use case in a way that is complementary with the Federal Data Strategy approach.
- [ ] Include other federated data use cases identified as possible priorities by strategy.data.gov.
- [ ] Group use cases based on similarities of implementation difficulty and ordered by magnitude of challenge.
- [ ] Based on grouping, determine which use cases would be the highest impact and best fit for our team.
- [ ] Set expectations about the type and amount of work we are able to do, then reach out to our top use cases.
- [ ] Evaluate how ready the top use cases are to work with our team. Better understand the challenges of the cases which are least able to work with us for future reference.
- [ ] Begin working with our top use case according to the expectations set.

**Comment by ethanheppner** [comment]
Closing this issue in favor of #129 and #130 . Will likely create a new issue to target selecting a second use case for the Django Data Ingest tool specifically once #129 is complete.
